app/jobs/article_attachments_upload_job.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+class ArticleAttachmentsUploadJob < ApplicationJob
+  queue_as :default
+
+  def perform(attachment, current_user_id)
+    ActiveRecord::Base.transaction do
+      article = Article.create!(user_id: current_user_id)
```

Comment:
create service object for controller attachment upload action and move db record creation and ArticleAttachmentsUploadJob creation there